arviz/stats/stats.py

```diff
@@ -587,6 +587,8 @@ def summary(data, var_names=None, include_circ=None, stat_funcs=None,
         Names of variables to include in summary
     include_circ : bool
         Whether to include circular statistics
+    fmt : {'wide', 'long', 'xarray'}
+        Return format is either DataFrame {'wide', 'long'} or xarray.Dataset {'xarray'}.
```

Comment:
I'm being so incredibly picky here
xarray.Dataset is prefixed with library, for consistency this could be pandas.DataFrame

Doesn't block a merge